releases.json

```diff
@@ -2433,5 +2433,13 @@
       "1.3.3-2",
       "1.3.3-1"
     ]
+  },
+  "boost-ext-di": {
+    "dependency_names": [
+      "di"
```

Comment:
The meson.build produces a pkg-config file for `boost_di`.

From a quick look, the CMake files don't appear to install anything under any name at all.

Reply:
I seem to have confused `dependency_names` and `program_names`. The wrap file provides the `di` dependency, so it should have been in `program_names`. Correctly?

Reply:
No. Program names is for when the project creates an `executable()` that can then be used as a command in e.g. a custom_target. :)

The question I have here really is just about what the "proper" name is. do we call the dependency "di"? Do we call it "boost_di"?

Reply:
Now I understand the question, thx for the explanation :)
I think since `di` is not part of `boost` yet, it's better to name the dependency `boost-ext-di`. Should I change the name in the files?

Reply:
The name in the WrapDB should always match whatever upstream uses.

Reply:
Yes, now I understand. In the `meson.build` file in the project, the name is `boost_di`.

Boost.di [meson.build](https://github.com/boost-ext/di/blob/cpp14/meson.build) file:
```Meson
project('boost_di', 'cpp', 
  version: '1.1.0',
  license: 'boost',
  default_options : ['warning_level=3', 'cpp_std=c++14']
)

boost_di_dep = declare_dependency(
  include_directories: include_directories('include', 'extension/include')
)

if not meson.is_subproject()
  install_headers('include/boost/di.hpp', subdir: 'di')
  
  pkgc = import('pkgconfig')
  pkgc.generate(name: 'boost_di',
    version: meson.project_version(),
    description: 'C++14 Dependency Injection Library'
  )
endif
```

I'll fix it now.